src/main.rs

```diff
@@ -75,53 +63,35 @@ struct Newer {
 
 impl Newer {
     fn new(args: Vec<String>) -> Result<Newer, Error> {
-        enum State {
-            Output,
-            Input,
-            Command,
-        }
+        let mut args = args.iter();
+
+        let newest = args
+            .by_ref()
+            .take_while(|&a| a != ":")
+            .map(|arg| match find_newest(arg) {
+                Err(e) if e.kind() == ErrorKind::NotFound => Ok(SystemTime::UNIX_EPOCH),
```

Comment:
This seems to be dropping the other errors?

Reply:
[See my following comment.](https://github.com/alecthomas/mk/pull/3/files#r1531444064)